src/components/ProductDetails.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import Button from './Button';
+import { propsPD } from '../../src/globalTypes';
+
+const ProductDetails = (props: propsPD) => {
+
+    const [picMain, setPicMain] = useState(0);
+
+    const urlSearchParams = new URLSearchParams(window.location.search);
+    const params = Object.fromEntries(urlSearchParams.entries());
+    const prod = props.products[3];
+
+
+    return (
+        <div className='product-details'>
+            <div className='link-navigation'>
+                {`STORE >> ${prod.category.toUpperCase()} >> ${prod.brand.toUpperCase()} >> ${prod.title.toUpperCase()}`}
+            </div>
+            <div className='product-item'>
+                <div className='item-title'>
+                    {prod.title}
+                </div>
+                <div className='product-description'>
+                    <div className='pics-block'>
+                        <div className='pics'>
+                            <div
+                                className='pic-1'
```

Comment:
Again creation of similar elements. The special function for it is the good option.